conmon/conmon.c

```diff
@@ -156,6 +156,13 @@ static int log_fd = -1;
 		exit(EXIT_FAILURE); \
 	} while (0)
 
+#define pwarn(s) \
+	do { \
+		fprintf(stderr, "[conmon:e]: %s %s\n", s, strerror(errno)); \
+		if (opt_syslog) \
+			syslog(LOG_ERR, "conmon %.20s <error>: %s %s\n", opt_cid, s, strerror(errno)); \
```

Comment:
Same here, `LOG_WARN` and `<pwarn>` instead.

Reply:
There is no `LOG_WARN` so using `LOG_INFO` instead. Thanks!